mk/spksrc.common.mk

```diff
@@ -54,7 +54,7 @@ AVAILABLE_TCVERSIONS = $(sort $(foreach arch,$(AVAILABLE_TOOLCHAINS),$(shell ech
 # Available toolchains formatted as '{ARCH}-{TC}'
 AVAILABLE_KERNEL = $(subst syno-,,$(sort $(notdir $(wildcard $(BASEDIR)kernel/syno-*))))
 AVAILABLE_KERNEL_VERSIONS = $(sort $(foreach arch,$(AVAILABLE_KERNEL),$(shell echo ${arch} | cut -f2 -d'-')))
-SUPPORTED_KERNEL_VERSIONS = 6.2.4 7.0
+SUPPORTED_KERNEL_VERSIONS = 6.2.4 7.1 7.2
```

Comment:
I don't know whether it is related to this line,
but when you run `make all-supported` then the x64-7.1 package builds and includes the 7.1 and 7.2 kernel modules.

Reply:
That is by design so a single 7.x package can serve them all.  Although I'm not sure this is the most effective method and wonder if it shouldn't only be version specific.

Reply:
I've reverted this behavior to build only called-up DSM #